Package.swift

```diff
@@ -1,10 +1,8 @@
-// swift-tools-version:5.3
-// In order to support users running on the latest Xcodes, please ensure that
-// Package@swift-5.5.swift is kept in sync with this file.
+// swift-tools-version:5.5
```

Comment:
I think it is safe for us to bump to use Swift 5.5 (Released on SEPTEMBER 20, 2021)